storage/innobase/include/srv0start.h

```diff
@@ -188,4 +188,7 @@ void srv_fatal_error() MY_ATTRIBUTE((noreturn));
 Shutdown all background threads created by InnoDB. */
 void srv_shutdown_all_bg_threads();
 
+dberr_t check_mk_and_keyring_encrypt_exclusion_for_undo(
```

Comment:
Please add srv_ prefix for all functions introduced in srv0*
add doxygen comments